time start_time
time end_time
bool upper_bound  # upper bound confidence of people rate 
bool scale        # number of observations has been done
# (can not both upper_bound and scale be true)
---
string[] region_ids
float64[] estimates
